x-pack/plugins/canvas/canvas_plugin_src/functions/common/ply.ts

```diff
@@ -140,8 +104,57 @@ export function ply() {
           type: 'datatable',
           rows,
           columns,
-        };
+        } as Datatable;
       });
     },
   };
 }
+
+function combineColumns(arrayOfColumnsArrays: DatatableColumn[][]) {
```

Comment:
Probably not necessary to change here, but retaining the order of definitions would help comparing changes eg. that of `combineColumns`